nautobot/ui/src/components/ObjectListTable.js

```diff
@@ -174,6 +181,13 @@ export default function ObjectListTable({
                 mt="3"
                 isLoaded={data_loaded}
             >
+                {!data_fetched && data_loaded ? (
+                    <Box background="white-0" borderRadius="md" padding="md">
+                        <LoadingWidget name={tableTitle}></LoadingWidget>
```

Comment:
```suggestion
                        <LoadingWidget name={tableTitle} />
```

Reply:
why does the `LoadingWidget` needs the name props?

Reply:
`tableTitle` in this case is a capitalized name of the models. So for Locations, the widget would display `Loading Locations ...`. For IPAddresses, it would display `Loading Ip Addresses ...`.